README.md

```diff
@@ -164,6 +166,7 @@ nnoremap gas :lua require('textcase').current_word('to_snake_case')<CR>
 nnoremap gad :lua require('textcase').current_word('to_dash_case')<CR>
 nnoremap gan :lua require('textcase').current_word('to_constant_case')<CR>
 nnoremap gad :lua require('textcase').current_word('to_dot_case')<CR>
+nnoremap gac :lua require('textcase').current_word('to_comma_case')<CR>
```

Comment:
This "conflicts" with the camel case mapping. Why not use `ga,`?

Reply:
I think this is just what I went with in my fork since the things I was working on at the time were more `to_comma_case` transformations and little to no `to_camel_case` transformations.

To your point though, `ga,` is certainly a more sensible plugin default for `to_comma_case`.